__tests__/__unit__/dataset/actions.unit.test.ts

```diff
@@ -0,0 +1,1508 @@
+import * as vscode from "vscode";
+import * as zowe from "@zowe/cli";
+import {
+    generateBasicZosmfSession, generateInstanceOfProfile,
+    generateIProfile,
+    generateISession, generateISessionWithoutCredentials, generateTextDocument,
+    generateTreeView
+} from "../../../__mocks__/mockCreators/shared";
+import {
+    generateDatasetAttributes,
+    generateDatasetSessionNode,
+    generateDatasetTree
+} from "../../../__mocks__/mockCreators/datasets";
+import { ZoweDatasetNode } from "../../../src/dataset/ZoweDatasetNode";
+import { bindMvsApi, generateMvsApi } from "../../../__mocks__/mockCreators/api";
+import * as dsActions from "../../../src/dataset/actions";
+import * as globals from "../../../src/globals";
+import * as path from "path";
+import * as fs from "fs";
+import * as sharedUtils from "../../../src/shared/utils";
+import { Profiles } from "../../../src/Profiles";
+
+// Missing the definition of path module, because I need the original logic for tests
+jest.mock("fs");
+
+let mockClipboardData = null;
```

Comment:
Can we move this line into createGlobalMocks? So that mockClipboardData is reset to null before every run.